src/screens/EncounterHomeScreen.tsx

```diff
@@ -39,8 +38,8 @@ const a11yLabels = consts.a11yLabel;
 // eslint-disable-next-line complexity
 export const EncounterHomeScreen = ({ navigation, route }: any) => {
   // @ts-expect-error settings are not properly typed
-  const encounterCategoryId = useContext(SettingsContext).globalSettings?.settings?.encounter
-    ?.categoryId;
+  const encounterCategoryId =
+    useContext(SettingsContext).globalSettings?.settings?.encounter?.categoryId;
```

Comment:
https://github.com/smart-village-solutions/smart-village-app-app/commit/ffb3f412f5aed4e868fbe512cea38d0b67426c1e

Reply:
I'm sorry, but I don't know what to do. I'd like you to show me what to do when you have time. 🙏

Reply:
the linter should not argue about the code. when you save and format to two lines because one line is too long, the ts-error comment does not match the line anymore. this needs to be fixed or ignored with reason or anything.

Reply:
b78bf57